.devilbox/www/include/lib/container/Memcd.php

```diff
@@ -26,13 +26,12 @@ class Memcd extends BaseClass implements BaseInterface
 	 *
 	 *********************************************************************************/
 
-	/**
-	 * Use singleton getInstance() instead.
-	 *
-	 * @param string $user Username
-	 * @param string $pass Password
-	 * @param string $host Host
-	 */
+    /**
+     * Use singleton getInstance() instead.
+     *
+     * @param $hostname
```

Comment:
:+1: thanks for fixing. Please consider using tabs instead of spaces to stick to the same coding standard